device/components/RTA_OnDeviceComponentTask.brs

```diff
@@ -39,35 +31,107 @@ sub runTaskThread()
 		"deleteRegistrySections": {
 			"handler": processDeleteRegistrySectionsRequest
 		}
+		"fileSystemGetVolumeList": {
```

Comment:
The term `fileSystem` is a bit verbose. Is it important to actually prefix these with `fileSystem` at all? What about these names instead?:

- `fileSystemGetVolumeList` -> `getVolumes` 
- `fileSystemGetDirectoryListing` -> `listDirectories` or `listDirectory` or `readdir` 
- `fileSystemStat` -> `fsStat` 
- `fileSystemCreateDirectory` -> `makeDirectory`
- `fileSystemDelete` -> `deletePath` or `fsRemove`
- `fileSystemRename` -> `renamePath` or `fsRename` 
- `readFile` -> `readFile`
- `writeFile` -> `readFile`

Reply:
I was trying to make the naming match the underlying functions. Some like `delete` weren't specific enough to make it specific to files. I'd be ok with `fs` prefix but would prefer to keep naming similar to the Roku names vs trying to make it similar to js libraries

Reply:
OH, I didn't realize they were named the same as the actual roku function names. if that's the case, then yeah, keep them the same name.

Reply:
After our discussion I've updated based off what we talked about